mainnet-contracts/src/L1RewardManager.sol

```diff
@@ -110,6 +114,15 @@ contract L1RewardManager is
             revert NotAllowedMintFrequency();
         }
 
+        bytes32 intervalId = keccak256(abi.encodePacked(params.startEpoch, params.endEpoch));
+
+        // Check duplicate interval ID
+        if ($.usedIntervalIds[intervalId]) {
+            revert DuplicateIntervalId(intervalId);
+        }
+
+        $.usedIntervalIds[intervalId] = true;
```

Comment:
lzCompose is missing delete of this. If the interval is reverted, the same interval can't be repeated.
It is unlikely that we would repeat the same exact interval, but still possible. Do we care about that?